sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 ## 1.0.0-preview.4 (Unreleased)
 
-### Breaking changes
+### Renames
+- Property `RequestedOn` renamed to `TrainingStartedOn` on `CustomFormModel` and `CustomFormModelInfo`.
+- Property `CompletedOn` renamed to `TrainingCompletedOn` on `CustomFormModel` and `CustomFormModelInfo`.
+- Parameter `formUrl` in `StartRecognizeContent` has been renamed to `formUri`.
+- Parameter `receiptUrl` in `StartRecognizeReceipts` has been renamed to `receiptUri`.
```

Comment:
More like nitpicking, but should we rename Url => Uri in our samples as well?
https://github.com/Azure/azure-sdk-for-net/blob/59625b45dfdd129e90c59ff95c3944d742a26c5b/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_CopyModel.cs#L20

Reply:
I went and actually changed the type of the strings that could actually be Uri.
For this specific one, it is the URL to the blob storage, we haven't converted it into an Uri so left it as is